### ISSUE TITLE
Zoom resets after compare

### ISSUE THREAD

**Issue by matework** [jsleroy/compare-plugin#82]
I've noticed that after I compare two files in Notepad++, the zoom of all opened files resets.

I'm using Notepad++ v7.4.2 (32-bit) on Windows 10
I'm using Compare Plugin v2.0.0

**Comment by pnedev** [comment]
Yes, the zoom of the first file is applied to the second one so their views are synchronized (for better diffs aligning). After the compare is closed the zoom should be restored.
What is the behavior you would expect?

**Comment by matework** [comment]
The behaviour you describe sounds good, but it's not what I experience.

If I zoom in on one tab, the other tab also zooms in, so it's not an issue of having each tab at different zoom levels. When I compare the two tabs, both zoom out to what I assume is the default zoom level. Once I'm finished the comparison, all tabs remain at the default zoom level. They do not reset to the zoom level from before the comparison.

**Comment by pnedev** [comment]
> They do not reset to the zoom level from before the comparison.

That sounds like an issue. I'll have a look, thanks for reporting.

BR

**Comment by sky5walk** [comment]
Similar...
My zoom changes to a smaller scale after closing Compare.
Compare Plugin v2.0.0 on Notepad++ v7.5.1.
